telegram/ext/_callbackcontext.py

```diff
@@ -155,7 +155,9 @@ def bot_data(self) -> BD:
     @bot_data.setter
     def bot_data(self, value: object) -> NoReturn:
         raise AttributeError(
-            "You can not assign a new value to bot_data, see https://git.io/Jt6ic"
+            "You can not assign a new value to bot_data, see"
+            " https://github.com/python-telegram-bot/python-telegram-bot"
+            "/wiki/Storing-bot%2C-user-and-chat-related-data"
```

Comment:
Can you put the link in a module-level variable - e.g. `_STORING_DATA_WIKI` - and insert it using f-strings here and below? makes it a bit easier to manage in case we need to change the link or add new errors with similar text :)

Reply:
Please verify my new commit [3c00950]